test/remote/gateways/remote_litle_test.rb

```diff
@@ -78,6 +102,14 @@ def test_successful_authorization_with_merchant_data
       merchant_grouping_id: 'brilliant-group'
     )
     assert response = @gateway.authorize(10010, @credit_card1, options)
+  end
+
+  def test_successful_authorization_with_echeck
+    options = @options.merge({
+      order_id: '38',
+      order_source: 'telephone'
+    })
+    assert response = @gateway.authorize(3002, @authorize_check, options)
     assert_success response
     assert_equal 'Approved', response.message
   end
```

Comment:
I know Github can do weird things to how these diffs are displayed, but wanted to make sure that these lines 113 and 114 weren't missing from the above test, and same for 202/203

Reply:
Ooh look like it is. I think it was a merge conflict auto-merging thing. Will fix that.